ilc/server/tailor/configs-injector.js

```diff
@@ -149,7 +150,16 @@ module.exports = class ConfigsInjector {
         const apps = _.mapValues(registryConfig.apps, v => _.pick(v, ['spaBundle', 'cssBundle', 'dependencies', 'props', 'kind']));
         const spaConfig = JSON.stringify(_.omit({...registryConfig, apps}, ['templates']));
 
-        return `<script type="spa-config">${spaConfig}</script>`;
+        return `<script type="ilc-config">${spaConfig}</script>`;
+    };
+
+    #getIlcState = (request) => {
+        const state = request.ilcState || {};
```

Comment:
I don't understand when we would not have `ilcState`
Because I see `ilcState` is always expected here is ilc/server/tailor/error-handler.js and it would be added to the request here is ilc/server/app.js